Global icons (#201)
springernature/frontend-global-toolkit: packages/global-context/README.md

	- [Color](#color)
	- [Foreground color](#foreground-color)
	- [Layers](#layers)

> - [Icons](#inons)

Reviewer: ```suggestion
- [Icons](#icons)
```

z-index: z("layer-name", "layer-variant", $my-map); //return z-index a name and variant from $my-map map
```

## Icons

> A set of icons is provided for use across all brands in the /img/icons folder. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 

Reviewer: Maybe link `img/icons folder` to the folder?

Author: Love to, how?

Reviewer: I think the path is right in my suggest.  Can check once it's added in.

> A set of icons is provided for use across all brands in the /img/icons folder. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 

Reviewer: ```suggestion
A [set of icons](img/icons) is provided for use across all brands. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 
```

> There is a recommendation that SVG Icons are inlined, but this is a choice.

Reviewer: ```suggestion
There is a recommendation that [SVG sprites](https://css-tricks.com/svg-sprites-use-better-icon-fonts/) are used, but this is a choice. The sprite can be made manually or if you use Gulp then they can be made automatically with [SVG Store](https://www.npmjs.com/package/gulp-svgstore).
```

> There is a recommendation that SVG Icons are inlined, but this ia a choice.

Reviewer: ```suggestion
There is a recommendation that SVG Icons are inlined, but this is a choice.
```

> For further information on the various implemenation methods, please read this article https://css-tricks.com/pretty-good-svg-icon-system/

Reviewer: ```suggestion
For further information on the various implementation methods, please read this article https://css-tricks.com/pretty-good-svg-icon-system/
```

> The simplest way is to just inline them, as detailed in this article https://css-tricks.com/pretty-good-svg-icon-system/

Reviewer: ```suggestion
```
